src/api/flaskr/service/shifu/shifu_mdflow_funcs.py

```
@@ -0,0 +1,33 @@
+from flask import Flask
+from flaskr.service.shifu.models import DraftOutlineItem
+from flaskr.service.common import raise_error
+from flaskr.dao import db
+
+
+def get_shifu_mdflow(app: Flask, shifu_bid: str, outline_bid: str) -> str:
```

Ruff (0.13.1)

Unused function argument: `shifu_bid`

(ARG001)